js/content.js

```diff
@@ -30,9 +30,10 @@ function sum (previousValue, currentValue) {
   }
 }
 
-// get the id of the page from the link of the profile picture change url
-
-id = $("div.a-popover-preload div.profile a.a-declarative").attr("href").split("/")[7].split("&")[2].split("=")[1]
+// get wishlist attributes for the currently active wishlist
+var $wishlist_link = $(".g-active-list a");
```

Comment:
Well, this broke *again* (this is what was broken when I originally opened the PR). This is really brittle and restating the name that we can see clearly on the page isn't a terribly valuable feature, so I think I'm going to take it out.